Makefile

```diff
@@ -0,0 +1,19 @@
+SHELL := /bin/bash
```

Comment:
May be preferable to use `/usr/bin/env bash` as this ensures if there is more than one bash installed, that the one in the PATH is used (this is a common case if you're using darwin).

Reply:
Can we leave `/usr/bin/env` out of it and just use `SHELL := bash`, relying on `$PATH`? I'm not sure about Makefile behaviour in this way.